crates/python/Cargo.toml

```diff
@@ -40,6 +40,7 @@ once_cell = "1.18.0"
 opentelemetry = { version = "0.23.0" }
 opentelemetry_sdk = { version = "0.23.0" }
 tracing = { version = "0.1.37" }
+prost = "0.12.6"
```

Comment:
Possibly not necessary any more

Reply:
Good thing to check. However, we do still need prost to access the Message trait.